helm/hpcc/values.yaml

```diff
@@ -207,6 +233,9 @@ esp:
   replicas: 1
   public: true
   servicePort: 8510
+  #resources:
+  #  cpu: "2"
```

Comment:
trivial: not sure any of these esp services warrant 2 cpus vs e.g. myeclccserver having just 1.